replica/database.hh

```diff
@@ -1514,6 +1514,8 @@ public:
     const column_family& find_column_family(const table_id&) const;
     column_family& find_column_family(const schema_ptr&);
     const column_family& find_column_family(const schema_ptr&) const;
+    const column_family* find_column_family_if_exists(std::string_view ks_name, std::string_view cf_name, bool warn = true) const;
+    const column_family* find_column_family_if_exists(const table_id& uuid, bool warn = true) const;
```

Comment:
The default `warn = true` is repair-specific. General users may not want to log. 

Maybe make this helper local to repair as a static method.